src/components/dashboard/jumbotron/OngoingJumbotronContainer.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import { getActiveEvents } from '../../../redux/getters/getEvents';
+import OngoingJumbotron from './OngoingJumbotron';
+import { getSortedTrips } from '../../../redux/getters/getTrips';
+
+const mapStateToProps = (state, ownProps) => {
+  let activeEvents = getActiveEvents(state, ownProps.trip_id);
+
+  return {
+    currEvents: activeEvents.current,
+    upcomingEvents: activeEvents.upcoming,
+    trip: getSortedTrips(state).active.shift()
```

Comment:
What is this one? Is this getting the first upcoming event?

Reply:
The current trip. I'm going to make this a little nicer though :)